static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -1,4 +1,5 @@
-import { mountDynamic } from '@guardian/automat-modules';
+import React from 'react';
```

Comment:
This is actually react here right? As opposed to preact?

Reply:
good question, we are using preact now. I was able to get preact to work everywhere in frontend (we were only having some issues with AudioPlayer.js https://github.com/guardian/frontend/pull/23637) and now we alias everything to `preact/compat` https://github.com/guardian/frontend/pull/23643

I was thinking of unifying the patter to import for react everywhere to keep consistency

Reply:
Oh brilliant, nice work 👍  I'd missed those other PRs go by.